test/io-tests.sh

```diff
@@ -181,6 +181,31 @@ ensureAppSettings(){
   pgrStop
 }
 
+getSocketStatus() {
+  curl -sL -w "%{http_code}\\n" -o /dev/null localhost:54321
+}
+
+socketConnection(){
+  # map port 54321 traffic to unix socket as workaround for old curl version
+  # not supporting --unix-socket flag
+  ncat -vlk 54321 -c 'ncat -U /tmp/postgrest.sock' &
+  pgrStart "./configs/unix-socket.config"
+  while pgrStarted && test "$( getSocketStatus )" -ne 200
```

Comment:
If the change to `nc` is possible, maybe `test -ne` could fail, you could try `test "$( getSocketStatus )" != "200"`